go/vt/vttablet/tabletmanager/vreplication/vreplicator.go

```diff
@@ -78,8 +78,8 @@ type vreplicator struct {
 	state           string
 	stats           *binlogplayer.Stats
 	// mysqld is used to fetch the local schema.
-	mysqld    mysqlctl.MysqlDaemon
-	pkInfoMap map[string][]*PrimaryKeyInfo
+	mysqld     mysqlctl.MysqlDaemon
+	colInfoMap map[string][]*ColumnInfo
```

Comment:
`map[string][]*ColumnInfo` confuses me. Is the key a table-name? Let's consider making a new `type` to simplify the syntax.

Reply:
I sometimes use type aliases for this. 

```golang
type tableNameStr = string
colInfoMap := map[tableNameStr][]*ColumnInfo{}
``